Hotels/Controllers/HotelRoomsController.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Hotels.Models;
+using Hotels.Services;
+
+namespace Hotels.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class HotelRoomsController : ControllerBase
+    {
+        private readonly IHotelRoomRepository repository;
+
+        public HotelRoomsController(IHotelRoomRepository repository)
+        {
+            this.repository = repository;
+        }
+
+        // GET: api/HotelRooms
+        [HttpGet]
+        public async Task<IEnumerable<HotelRoom>> GetHotelRooms()
+        {
+            return await repository.GetAllAsync();
+        }
+
+        // GET: api/HotelRooms/5
+        [HttpGet("{id}")]
+        public async Task<ActionResult<HotelRoom>> GetHotelRoom(long id)
```

Comment:
We actually want this route, and the `PUT` and `DELETE` below, to be: `api/Hotels/{hotelId}/Rooms/{roomNumber}`